tools/SetupFlow/DevHome.SetupFlow/Services/ConfigurationFileBuilder.cs

```diff
@@ -36,21 +32,33 @@ public ConfigurationFileBuilder(SetupFlowOrchestrator orchestrator)
     /// <returns>The config file object representing the yaml file.</returns>
     public WinGetConfigFile BuildConfigFileObjectFromTaskGroups(IList<ISetupTaskGroup> taskGroups, ConfigurationFileKind configurationFileKind)
     {
-        var listOfResources = new List<WinGetConfigResource>();
-
+        List<WinGetConfigResource> repoResources = [];

```

Comment:
why `[]` and not `new ()`?

Reply:
It's a new syntax in C# 12 
Reference: [Collection Expressions](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-12#collection-expressions)